lib/sqlalchemy/ext/asyncio/session.py

```diff
@@ -231,6 +231,33 @@ async def scalar(
         )
         return result.scalar()
 
+    async def scalars(
+        self,
+        statement,
+        params=None,
+        execution_options=util.EMPTY_DICT,
+        bind_arguments=None,
+        **kw
+    ):
+        """Execute a statement and return scalar results.
+
```

Comment:
**mike bayer** (zzzeek) wrote:

add

    :return: an :class:`_result.ScalarResult` object

View this in Gerrit at https://gerrit.sqlalchemy.org/c/sqlalchemy/sqlalchemy/+/3065